packages/app/src/request.ts

```diff
@@ -16,16 +16,20 @@ export { getURLParams } from '@tinyhttp/req'
 
 const trustRemoteAddress = ({ socket }: Pick<Request, 'headers' | 'socket'>) => {
   const val = socket.remoteAddress
-  if (typeof val === 'string') return compile(val.split(',').map((x) => x.trim()))
+  if (typeof val === 'string') {
+    return compile(val.split(',').map((x) => x.trim()))
+    /* c8 ignore next */
+  }
   return compile(val || [])
 }
 
 export const getProtocol = (req: Request): Protocol => {
   const proto = `http${req.secure ? 's' : ''}`
 
+  /* c8 ignore next */
```

Comment:
don't use c8 ignores

Reply:
`trustRemoteAddress` always returns a truthy value. Not sure how to test it